### ISSUE TITLE
<Radio /> is type checkbox instead of radio

### ISSUE THREAD

**Issue by Radek-Palisa** [kiwicom/orbit-components#362]
## Expected Behavior
<Radio /> should behave like native html element radio in terms of keyboard navigation and accessibility.

## Current Behavior
Since the component is currently of `type="checkbox"` it doesn't behave as it should

## Possible Solution
change the type to type radio
